### ISSUE TITLE
APOD endpoint returns 504

### ISSUE THREAD

**Issue by JF631** [nasa/api-docs#197]
The apod endpoint currently returns `504 Gateway Time-out`

Following call is not working:
https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY

(Tried aswell using an official API key. However, this does not seem to solve the issue.)

**Comment by JulianusIV** [comment]
Yeah same here, happened today, after getting 503/502 the last days. There is something odd going on with the api this week

**Comment by KiwiSkunk** [comment]
Still ongoing. My wallpaper is looking dull...

**Comment by tbuchloh** [comment]
Yes, I can confirm. I also experience 504 since 2023-06-02.

**Comment by JF631** [comment]
It now worked for a couple of days.
However it returned to the `504 Gateway Time-out` state.

**Comment by sahaskamble** [comment]
did someone find any answer what is happening ?
